### PR TITLE
Fix format of generator docs [ci skip]

### PR DESCRIPTION
**Pull request by yhirano55**
### Summary

- Add typewriter style to lib method in `actions.rb`.
- ~Unify typewriter style to `++` in `actions.rb`.~
- ~Replace quote style with typewriter style in `app_base.rb`.~

### CODE REVIEW

**Comment by kamipo** [comment]
> - Unify typewriter style to `++` in `actions.rb`.

Why? `<tt>` doesn't work?

**Comment by yhirano55** [comment]
I thought it would be better that `+` style is shorter than `<tt>`.
But it's my mistake... I'll restore it, soon :bow:

File: railties/lib/rails/generators/app_base.rb [hunk 1]
```diff
@@ -377,7 +377,7 @@ def bundle_command(command)
         say_status :run, "bundle #{command}"
 
         # We are going to shell out rather than invoking Bundler::CLI.new(command)
-        # because `rails new` loads the Thor gem and on the other hand bundler uses
+        # because +rails new+ loads the Thor gem and on the other hand bundler uses
```
Comment (kamipo) [inline railties/lib/rails/generators/app_base.rb]: Actually this is a comment in the method, not a doc. It is no worth to change to rdoc format.

Comment (yhirano55) [inline railties/lib/rails/generators/app_base.rb]: fmm.., I understood it.

Comment (yhirano55) [inline railties/lib/rails/generators/app_base.rb]: @kamipo Thanks for your comment. I'll study more rdoc specifications.

**Pull request merged**